### DISCUSSION TITLE
多账户下没有域名的账号会提示失败

### DISCUSSION THREAD

**Discussion by jacobax** [luolongfei/freenom#29] [General]
不影响有域名的账号，没域名的号注册域名后重启正常显示成功
许久没看，之前的action版被删了，刚刚docker部署到vps成功运行，再次感谢作者大大

**Comment by luolongfei** [comment]
@Jacobax action 版本因为不符合 GitHub 的用户协议，之前被删库了，已经完全移除与 action 相关的内容了。

没域名报错的问题这边先标记了，后面空了加上判断。
